frontend-ssr/pages/submissions.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Head from 'next/head';
+import Footer from '~/components/footer/Footer.component';
+import Header from '~/components/header/Header.component';
+
+import ExSubmission from '../components/exercise-submissions/ExerciseSubmission';
```

Comment:
As a rule of thumb, always name the import exactly like the file name.  `ExerciseSubmissions`, `ExerciseSubmissionsSkeleton`, etc. Although this means a little more code, it helps avoid some confusion because everything is named the same way.

I don't have to ask myself: is `ExSubmission` same as `ExerciseSubmission` or a little different?